asapdiscovery-data/asapdiscovery/data/aws/cli.py

```diff
@@ -0,0 +1,108 @@
+from pathlib import Path
+
+import click
+
+from ..cli import cli
+
+# use these extra kwargs with any option from a settings parameter.
+SETTINGS_OPTION_KWARGS = {
+    "show_envvar": True,
+}
+
+
+def aws_params(func):
```

Comment:
Are any of these strictly required? I assume some are, but you are probably best place to know which. If so could you mark them as `required=True` to stop downstream failures.